src/Button/sizes.ts

```diff
@@ -0,0 +1,5 @@
+const SIZES = ['jumbo', 'large', 'medium', 'small'] as const;
+
+export type Size = typeof SIZES[number];
```

Comment:
Reminder that this syntax results in a union type like `'jumbo' | 'large' | 'medium' | 'small'`.